application/common/models/Verification.php

```diff
@@ -58,16 +60,57 @@ public static function sendEmail(
             $parameters
         );
 
-        EmailQueue::sendOrQueue(
-            $toAddress,
-            $subject,
-            $body,
-            $body,
-            $ccAddress,
-            $eventLogUserId,
-            $eventLogTopic,
-            $eventLogDetails
-        );
+        /*
+         * If configured to use external email service send through that instead of using
+         * local EmailQueue service
+         */
+        if (\Yii::$app->params['emailVerification']['useEmailService']) {
```

Comment:
You could check for the else condition first (which has a shorter code block) and have a return at the end of it and then not have this long if block indented.